modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -1611,6 +1612,53 @@ private void _updateTable(
 		}
 	}
 
+	private void _validateFileExtension(
```

Comment:
Could you test if this works for this validation? Let me know your opinion if you think it's worth changing.
```
ObjectFieldSetting objectFieldSetting =
	_objectFieldSettingPersistence.fetchByOFI_N(
		objectFieldId, "acceptedFileExtensions");

String objectFieldSettingValue = objectFieldSetting.getValue();

String[] acceptedFileExtensions = objectFieldSettingValue.split(
	"\\s*,\\s*");

if (!ArrayUtil.contains(acceptedFileExtensions, fileExtension)) {
	throw new ObjectEntryValuesException.InvalidFileExtension(
		fileExtension, objectFieldName);
}
```

Reply:
I applied the change, thanks!